src/Mailer.php

```diff
@@ -136,8 +136,16 @@ public function extractMessage(MailerMessage $message)
             'Sender' => $message->getSender(),
             'Cc' => $message->getCc(),
             'Bcc' => $message->getBcc(),
+            'DeduplicateCampaign' => $message->getDeduplicateCampaign(),
             'TemplateErrorReporting' => $message->getTemplateErrorReporting() ? $message->getTemplateErrorReporting() : $this->getDefaultTemplateErrorReporting(),
         ];
+
+        if ($message->getCustomCampaign() != ''){
```

Comment:
I just saw this now... I dunno why, but it should be the same as with getCustomId, we dont compare against empty strings `''`.

Reply:
Changed this.